cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -68,3 +73,41 @@ func ValidateTokenDiscovery(c *kubeadm.TokenDiscovery, fldPath *field.Path) fiel
 	allErrs := field.ErrorList{}
 	return allErrs
 }
+
+func ValidateCertificatesPhase(c *kubeadm.CertificatesPhase, fldPath *field.Path) field.ErrorList {
```

Comment:
Please add tests for these new functions in this PR based on the discussion during the SIG meeting.